src/transactions/utils/validation.js

```diff
@@ -47,3 +47,19 @@ export const validateKeysgroup = keysgroup => {
 	}
 	return validatePublicKeys(keysgroup);
 };
+
+// Specification: https://github.com/LiskHQ/lisk/blob/1.0.0/logic/account.js#L373
```

Comment:
`1.0.0` is an even less stable reference than `development`. If we're going to refer to a code line it should be via a commit. Preferably we'd have a documentation site though.

Reply:
Okay, removing this entirely